internal/winapi/jobobject.go

```diff
@@ -28,7 +28,7 @@ const (
 // https://docs.microsoft.com/en-us/windows/win32/procthread/job-object-security-and-access-rights
 const (
 	JOB_OBJECT_QUERY      = 0x0004
-	JOB_OBJECT_ALL_ACCESS = 0x1F001F
```

Comment:
where does this change come from? from the [docs](https://learn.microsoft.com/en-us/windows/win32/procthread/job-object-security-and-access-rights), it looks like there isn't a flag for 0x0020

Reply:
The docs may be a bit outdated. The extra delta comes from `JOB_OBJECT_IMPERSONATE`.